src/resources/templates/category_pages/liferay_dxp_7_1.hbs

```diff
@@ -0,0 +1,23 @@
+<div class="container-fluid container-fluid-max-xl main-content">
+	<div class="row">
+		<div class="col-12">
+			{{breadcrumbs}}
+		</div>
+	</div>
+
+	<div class="categories-main row">
+		<div class="category-heading col-md-12">
+			<h1>
+				{{dc '7_1_page_heading'}}
```

Comment:
Cole made a spreadsheet and updated all of the dynamic content. Since he's managing all dynamic contents, I'm using what he provided for now for testing purposes. But I also pointed out the discrepancy to him. 
https://docs.google.com/spreadsheets/d/1WsJ4ZLqZYsKtR64KCbmI82g5Gg7h9HVmBVjXTonRAhs/edit#gid=0

Reply:
Cole just updated the key, I will send you a new pr with the updated dynamic content.

Reply:
Thanks @rdai10!